File: sample/src/main/java/co/opensi/kkiapay_sdk/MyApp.java
package co.opensi.kkiapay_sdk;

import android.app.Application;

import co.opensi.kkiapay.uikit.Kkiapay;
import co.opensi.kkiapay.uikit.SdkConfig;

public class MyApp extends Application {
    @Override
    public void onCreate () {
        super.onCreate ();
        //Initialisation de l'API
        Kkiapay.init(this,
                "3425dc6035d711eca8f5b92f2997955b",
                new SdkConfig (R.raw.armoiries, R.color.colorPrimary, true));
    }
}

File: sample/src/main/java/co/opensi/kkiapay_sdk/kotlin/MyApp.kt
package co.opensi.kkiapay_sdk.kotlin

import android.app.Application
import co.opensi.kkiapay.uikit.Kkiapay
import co.opensi.kkiapay.uikit.SdkConfig
import co.opensi.kkiapay_sdk.R

class MyApp : Application() {
    override fun onCreate() {
        super.onCreate()
        //Initialisation de l'API
        Kkiapay.init(applicationContext,
                "<PUBLIC-API-KEY>",
            SdkConfig(themeColor = R.color.colorPrimary,
                imageResource = R.raw.armoiries,
                    /* set enableSandbox = false in case you are using live API Keys */
                    enableSandbox = false))
    }
}
